### ISSUE TITLE
Punkbuster

### ISSUE THREAD

**Issue by PepsimanRUS** [Shady7557/CoDUO-FoV-Changer#2]
Hi, bans anti-cheat punkbuster for your program?

**Comment by Shady7557** [comment]
I haven't tested it in a long time, but no, it shouldn't. If I remember correctly, it may show a warning or two, though it won't ban. I can't say 100% for sure though, so try it at your own risk.

**Comment by PepsimanRUS** [comment]
The same goes for the Steam version?

**Comment by Shady7557** [comment]
That is correct, this works fine with the retail and Steam copies of the game. I own both, and I have confirmed it to work on both. If you have issues with it, let me know. I have made some changes locally that I have not pushed here on GitHub yet.

**Comment by PepsimanRUS** [comment]
Perform a punkbuster test in the Steam version.

**Comment by Shady7557** [comment]
If you're requesting me to perform a test on the Steam version, I will consider doing it soon. I will get back to you once I'm done.

**Comment by PepsimanRUS** [comment]
OK. Will wait.
